.azure/pipelines/Phonebook.preview.yml

```diff
@@ -0,0 +1,193 @@
+pr: 
+  branches:
+    include:
+      - master
+
+variables:
+  image_repo: danielhabenicht/phonebook
```

Comment:
is this a acr?

Reply:
I don't have push permission at the moment so we have to use my dockerhub account